po/pt_BR.po

```diff
@@ -0,0 +1,706 @@
+
+#: src/config.rs:740 src/config.rs:873
+msgid "value can not be empty for key '{}'"
+msgstr "valor para a chave '{}' não pode ser vazio"
+
+#: src/config.rs:756
+msgid "key '{}' does not belong to a section"
+msgstr "chave '{}' não pertence a secção"
+
+#: src/config.rs:762
+msgid "unknown section '{}'"
+msgstr "secção '{}' desconhecida"
+
+#: src/config.rs:769 src/config.rs:824 src/config.rs:829
+msgid "key can not be empty"
+msgstr "chave não pode ter um valor vazio"
+
+#: src/config.rs:791
+msgid "error: unknown option '{}' in section [bin]"
+msgstr "erro: opção desconhecida '{}' na secção [bin]"
+
+#: src/config.rs:891
+msgid "error: unknown option '{}' in section [options]"
+msgstr "erro: opção desconhecida '{}' na secção [options]"
+
+#: src/config.rs:896
+msgid "option '{}' does not take a value"
+msgstr "a opção '{}' não aceita valores"
+
+#: src/config.rs:947 src/install.rs:1796
+msgid "There are {:n} providers available for {:pkg}:"
+msgstr "Há {:n} provedores disponíveis para {:pkg}:"
+
+#: src/config.rs:961 src/info.rs:56 src/info.rs:95 src/install.rs:1755
+#: src/install.rs:1804
+msgid "Repository"
+msgstr "Repositório"
+
+#: src/config.rs:983
+msgid "  failed to sync {}"
+msgstr "  erro ao sincronizar '{}'"
+
+#: src/devel.rs:129 src/download.rs:179
+msgid "Querying AUR..."
+msgstr "Buscando no AUR..."
+
+#: src/devel.rs:142 src/devel.rs:166 src/install.rs:694
+msgid "failed to parse srcinfo for '{}'"
+msgstr "erro ao analisar srcinfo para '{}'"
+
+#: src/devel.rs:184
+msgid "Looking for devel repos..."
+msgstr "Procurando por repositórios devel..."
+
+#: src/devel.rs:193
+msgid "failed to save devel info"
+msgstr "erro ao salvar informações de desenvolvimento"
+
+#: src/devel.rs:382
+msgid "package does not have an update"
+msgstr "pacote não possui atualizações"
+
+#: src/devel.rs:417
+msgid "failed to lookup: {}"
+msgstr "falha ao buscar: {}"
+
+#: src/devel.rs:445
+msgid "invalid json: {}"
+msgstr "json inválido: {}"
+
+#: src/download.rs:83
+msgid "packages not in the AUR: "
+msgstr "pacotes não presentes no AUR: "
+
+#: src/download.rs:94
+msgid "marked out of date: "
+msgstr "desatualizados: "
+
+#: src/download.rs:105
+msgid "orphans: "
+msgstr "órfãos: "
+
+#: src/download.rs:206
+msgid "could not get current directory"
+msgstr "não foi possível obter o diretório atual"
+
+#: src/download.rs:210
+msgid "can not get repo packages: asp is not installed"
+msgstr "não foi possível obter pacotes dos repositórios: asp não está instalado"
+
+#: src/download.rs:231
+msgid "Missing ABS packages "
+msgstr "Pacotes ABS faltando "
+
+#: src/download.rs:250 src/download.rs:276 src/download.rs:530
+#: src/download.rs:543 src/download.rs:557 src/exec.rs:72 src/exec.rs:113
+#: src/exec.rs:122 src/exec.rs:148 src/exec.rs:161 src/exec.rs:186
+#: src/exec.rs:200 src/exec.rs:221 src/exec.rs:234 src/install.rs:744
+#: src/install.rs:807 src/install.rs:845 src/keys.rs:74
+msgid "failed to run:"
+msgstr "erro ao rodar:"
```

Comment:
See https://github.com/Morganamilo/paru/pull/435#discussion_r660053767

Reply:
Fixed that